src/core/gui/toolbarMenubar/ToolMenuHandler.cpp

```diff
@@ -492,8 +492,9 @@ void ToolMenuHandler::initToolItems() {
     addCustomItemTgl("DRAW_SPLINE", ACTION_TOOL_DRAW_SPLINE, GROUP_RULER, false, "draw-spline", _("Draw Spline"));
 
     addCustomItemTgl("SELECT_REGION", ACTION_TOOL_SELECT_REGION, GROUP_TOOL, true, "select-lasso", _("Select Region"));
-    addCustomItemTgl("SELECT_RECTANGLE", ACTION_TOOL_SELECT_RECT, GROUP_TOOL, true, "select-rect",
-                     _("Select Rectangle"));
+    addCustomItemTgl("SELECT_RECTANGLE", ACTION_TOOL_SELECT_RECT, GROUP_TOOL, true, "select-rect", _("Select Rectangle"));
+    addCustomItemTgl("SELECT_MULITLAYER_REGION", ACTION_TOOL_SELECT_MULTILAYER_REGION, GROUP_TOOL, true, "select-lasso", _("Select Multi-Layer Region"));
+    addCustomItemTgl("SELECT_MULITLAYER_RECT", ACTION_TOOL_SELECT_MULTILAYER_RECT, GROUP_TOOL, true, "select-lasso", _("Select Multi-Layer Rect"));
```

Comment:
There are still some mistakes:
```suggestion
    addCustomItemTgl("SELECT_RECTANGLE", ACTION_TOOL_SELECT_RECT, GROUP_TOOL, true, "select-rect", _("Select Rectangle"));
    addCustomItemTgl("SELECT_MULTILAYER_REGION", ACTION_TOOL_SELECT_MULTILAYER_REGION, GROUP_TOOL, true, "select-multilayer-lasso", _("Select Multi-Layer Region"));
    addCustomItemTgl("SELECT_MULTILAYER_RECT", ACTION_TOOL_SELECT_MULTILAYER_RECT, GROUP_TOOL, true, "select-multilayer-rect", _("Select Multi-Layer Rect"));
```

Reply:
Should I add the `SELECT_MULTILAYER_RECTANGLE`/`REGION` in this PR? Or should I maybe open a PR synchronizing the `TOOLBAR_INI_HEADER` with the toolbar.ini comment all together  (including the `SELECT_OBJECT`, `PLAY_OBJECT`,...)?